### ISSUE TITLE
When is next release scheduled to fix the "AttributeError: 'function' object has no attribute '__func__'" error in cache.py?

### ISSUE THREAD

**Issue by vignirvignir** [jazzband/django-debug-toolbar#1775]
TLDR; When is the next release scheduled?

This error is present in version 4.0.0 when making any request on an endpoint using django rest framework.
I see it has been probably fixed in the latest commit  https://github.com/jazzband/django-debug-toolbar/commit/a6b65a7c509589a635d5c3889ee8ea7a1e10b707

                | Traceback (most recent call last):
                |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
                |     response = get_response(request)
                |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/sentry_sdk/integrations/django/middleware.py", line 176, in __call__
                |     return f(*args, **kwargs)
                |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/debug_toolbar/middleware.py", line 64, in __call__
                |     panel.disable_instrumentation()
             |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/debug_toolbar/panels/cache.py", line 217, in disable_instrumentation
             |     self._unmonkey_patch_cache(cache)
             |   File "/opt/pysetup/.venv/lib/python3.9/site-packages/debug_toolbar/panels/cache.py", line 171, in _unmonkey_patch_cache
             |     if original_method.__func__ == getattr(cache.__class__, name):
             | AttributeError: 'function' object has no attribute '__func__'

**Comment by matthiask** [comment]
I do not have specific plans. Probably some time after #1773 has been merged (I'd really like tests to be green again)

**Comment by matthiask** [comment]
Fixed in 4.1.
